search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -216,7 +213,8 @@ function setPopularityTimeline() {
               'Popularity Timeline is not available for the selected country.';
         } else {
           drawPopularityTimeline(
-              timelineJSON.default.timelineData, popularityTimelineElement, 
+              timelineJSON.default.timelineData, 
+              popularityTimelineElement, 
```

Comment:
```suggestion
              timelineJSON.default.timelineData,
              popularityTimelineElement,
```
Avoid trailing whitespace.

Reply:
Beyond where required by the language or other style rules, and apart from literals, comments, and JSDoc, a single internal ASCII space also appears in the following places only.

After a comma (,) or semicolon (;). Note that spaces are never allowed before these characters.
https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace

Reply:
But at the start of that same section, it mentions trailing white space is forbidden. I am not sure which rule should be followed here. @hayleejane3  @lawrencezjh @ChenyuHeidiZhang  what do you guys think?

"Use of horizontal whitespace depends on location, and falls into three broad categories: leading (at the start of a line), trailing (at the end of a line), and internal. Leading whitespace (i.e., indentation) is addressed elsewhere. Trailing whitespace is forbidden."

Reply:
the beyond part lsists the exceptions to the use of horizontal whitespace rule